src/pages/homepage/homepage.js

```diff
@@ -1,18 +1,34 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './homepage.css';
 import '../../components.css';
-import { Link } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
+import { useFilter } from '../../context/filterContext';
+import axios from 'axios';
 
 export default function Homepage() {
+
+    const [categories, setCategories] = useState();
+
+    useEffect(async () => {
+        const res = await axios.get("/api/categories");
+        console.log(res);
+    })
+
+    const { state, dispatch } = useFilter();
+    const navigate = useNavigate();
+
+    const handleCategory = (e) => {
+        dispatch({ type: "CLEAR" });
+        dispatch({ type: "ADD_FILTER_BY_CATEGORY", payload: e });
+        navigate("/productlist");
+    }
     return (
         <div>
             <div className="homepage">
                 <div className="img-grid">
-                    <img alt="clothing" src="https://picsum.photos/id/1005/250/150" />
-                    <img alt="clothing" src="https://picsum.photos/id/1005/250/150" />
-                    <img alt="clothing" src="https://picsum.photos/id/1005/250/150" />
-                    <img alt="clothing" src="https://picsum.photos/id/1005/250/150" />
-                    <img alt="clothing" src="https://picsum.photos/id/1005/250/150" />
+                    <img className="category-img" onClick={() => handleCategory("womens clothing")} alt="clothing" src="https://images.unsplash.com/photo-1503342217505-b0a15ec3261c?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80" />
```

Comment:
`alt` text can be more descriptive than 'clothing'  

I would suggest to give category name as the `alt` text

Reply:
done this, thanks.